CONTRIBUTING.md

```diff
@@ -64,15 +64,9 @@ packages
         └─index.native.js
 ```
 
-### Theme
+### Theming
 
-Currently, we have 3 predefined themes `Corporate`, `EndUser` and `Gyms`, all
-of them follows the `BaseTheme` object.
-
-Every component has its own `theme` file. You can find it in its own folder.
-
-When building a new component, make sure to add values like paddings, colors,
-margins, to the component theme file.
+Yoga exposes a `createTheme` function in order to generate new themes, you can refer to [Theme Generator](https://gympass.github.io/yoga/components/theming/themeGenerator) in order to see how to create custom themes.
```

Comment:
```
Every component has its own `theme` file. You can find it in its own folder.

When building a new component, make sure to add values like paddings, colors,
margins, to the component theme file.
```
Is this not true anymore?

If it still is, I'd keep this info

Reply:
What a pair of hawk eyes! Thank you a lot, indeed I mistakenly removed more than needed